src/Kiota.Builder/Refiners/PythonRefiner.cs

```diff
@@ -20,7 +20,7 @@ public override Task Refine(CodeNamespace generatedCode, CancellationToken cance
                 _configuration.UsesBackingStore,
                 static s => s,
                 true,
-                $"{SerializationModuleName}.composed_type_wrapper",
+                $"{SerializationModuleName}",
```

Comment:
Just to confirm here, this is intentionally fixing a package name issue?

Reply:
Resolving after offline explanation that  change  in `kiota_abstractions/serialization/__init__.py` resolves this in https://github.com/microsoft/kiota-abstractions-python/pull/219/files